src/com/company/Field.java

```diff
@@ -0,0 +1,68 @@
+package com.company;
+
+public class Field {
+
+    private int[][] array = new int[10][10];
+    private int squareX;
+    private int squareY;
+    private int randomMap;
+
+    public Field(Point coordinates) {
+        this.squareX = coordinates.getSquareX();
+        this.squareY = coordinates.getSquareY();
+        this.createRandomMap();
+    }
+
+    public void createRandomMap() {
+        int firstMap = 1;
+        int lastMap = 3;
+        randomMap = firstMap + (int) (Math.random() * lastMap);
+        this.fillTab();
+    }
+
+    public int[][] getArray() {
+        return array;
+    }
+
+    private void fillTab() {
+        System.out.println(randomMap);
+        if (randomMap == 1) {
```

Comment:
Move generation of different random maps to different methods (method length should be 5-7 lines)